testsuite/oauth/refresh_token.py

```diff
@@ -46,13 +41,12 @@ def revoke_access_token(self):
         post_data = {
             'token': self.access_token,
         }
-        response = requests.delete(self._config['token_url'],
+        response = requests.delete(self._urls['token'],
```

Comment:
Still planning on it yeah. It's just hard to delete a code path that might eventually be useful :(